File: Pemrograman1/Modul1/Python/PRAK110-2310817310009-Muhamamd_Fauzan_Ahsani.py
import math

base = 5
height = 12
hypotenuse = math.sqrt(base**2 + height**2 )
hypotenuse = int(hypotenuse)  

print(f"Diketahui : \nAlas = {base} cm \nTinggi = {height} cm\n")
print(f"Jawab : \nSisi A = {height} cm\nSisi B = {base} cm\nSisi C = {hypotenuse} cm" )
print(f"Keliling = {base + height + hypotenuse} cm\nLuas = {int(.5 * base * height)} cm" )

File: Pemrograman1/Modul1/C/PRAK110-2310817310009-Muhamamd_Fauzan_Ahsani.c
#include <stdio.h>
#include <math.h>

void main(){
    double base = 5;
    double height = 12;
    double hypotenuse = sqrt(base*base + height*height ); 
  
    printf("Diketahui : \nAlas = %.0f cm \nTinggi = %.0f cm \n \n", base, height);
    printf("Jawab : \nSisi A = %.0f cm\nSisi B = %.0f cm\nSisi C = %.0f cm\n", height, base, hypotenuse);
    printf("Keliling = %.0f cm\nLuas = %.0f cm", base + height + hypotenuse, .5 * base * height );
}